File: testers/grpc/kotlin/generated-and-checked-in/com/example/grpc/PaymentMethodMethod.kt
package com.example.grpc



/** OneOf type for method */
sealed interface PaymentMethodMethod {
  data class BankTransferValue(val bankTransfer: BankTransfer) : PaymentMethodMethod

  data class CreditCardValue(val creditCard: CreditCard) : PaymentMethodMethod

  data class WalletValue(val wallet: Wallet) : PaymentMethodMethod
}

File: testers/grpc/scala-cats/generated-and-checked-in/com/example/grpc/PaymentMethodMethod.scala
package com.example.grpc



/** OneOf type for method */
sealed trait PaymentMethodMethod

object PaymentMethodMethod {
  case class BankTransferValue(bankTransfer: BankTransfer) extends PaymentMethodMethod

  case class CreditCardValue(creditCard: CreditCard) extends PaymentMethodMethod

  case class WalletValue(wallet: Wallet) extends PaymentMethodMethod
}